### PR TITLE
Add Ballerina Ambassador page

### LINKED ISSUE #9422: Create Ballerina Ambassador Program page

### PR DESCRIPTION
**Pull request by sm1990**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: _data/ambassadors.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (azinneera) [inline _data/ambassadors.json:1]: Are we showing this info anywhere?

File: components/ambassadors/apply/Apply.module.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+.sendEmail:hover {
+    background-color: #464646;
+    color: #fff !important;
+}
+
+@media (min-width: 992px) {
+    .sendEmail {
+        width: 50%;
+    }
+}
```
Comment (azinneera) [inline components/ambassadors/apply/Apply.module.css:21]:
```suggestion
}

```

File: components/ambassadors/meet/Meet.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+export default function Meet(props) {
+
+  const ambassadors = Ambassadors.ambassadors;
+
+  return (
+    <Col xs={12}>
+      <Container>
+        <Row>
+          <Col xs={12}>
+            <h2 id='meet-our-ambassadors' className='section'>
```
Comment (azinneera) [inline components/ambassadors/meet/Meet.js:35]: Is this section hidden? We don't have ambassadors yet.

File: _data/ambassadors.json (hunk 1 above)
Comment (sm1990) [reply _data/ambassadors.json:1]: No. The `Meet` component is currently hidden.

File: components/ambassadors/meet/Meet.js (hunk 3 above)
Comment (sm1990) [reply components/ambassadors/meet/Meet.js:35]: Yes

**Review by azinneera — APPROVED** [review]

**Pull request merged**